main/bind/APKBUILD

```diff
@@ -240,9 +275,17 @@ libs() {
 	depends="$depends_libs"
 	default_libs
 }
+_append_deps() {
+	local s split
+	for s in $subpackages; do
+		split="$(subpkg_set "$s"; printf -- '%s' "$subpkgsplit")"
+		eval "makedepends=\"\$makedepends \${depends_${split}}\""
```

Comment:
is there some way we can avoid use `eval`? eval is evil.

Reply:
I don't know of a safer way than to use `eval` here. If there is one, I am happy to learn about it.